src/main/java/com/fasterxml/jackson/databind/ObjectMapper.java

```diff
@@ -4789,6 +4789,26 @@ public void acceptJsonFormatVisitor(JavaType type, JsonFormatVisitorWrapper visi
         _serializerProvider(getSerializationConfig()).acceptJsonFormatVisitor(type, visitor);
     }
 
+    /*
+    /**********************************************************
+    /* Extended Public API: caches
+    /**********************************************************
+     */
+
+    /**
+     * Method that will clear all caches.
+     * This can be used to remove memory usage or to force re-construction cached entries after configuration changes.
```

Comment:
I'll change this a bit: second part is misleading -- clearing caches cannot/shouldn't be used to try to allow config changes for `ObjectMapper`: that (changing config after use) unsupported use case and not guaranteed to work (might work now, could break in future).